viriato_julia/REGK.jl

```diff
@@ -169,6 +194,7 @@ bperp_max = maximum(bperp)
 omega_kaw = omegakaw(bperp_max) # Function omega_kaw--its value is public in the function def
 
 # Calculate CFL fraction/timestep
+# TODO Viriato has terms with dz here, which I'm not sure are actually zero. Make sure!
```

Comment:
Is this TODO still relevant?

Reply:
Maybe, need to revisit this later, but 90% sure no change needed